kong/pdk/table.lua

```diff
@@ -52,10 +52,33 @@ do
 end
 
 
+--- Merges the contents of two tables together, producing a new one.
+-- The entries of both tables are copied non-recursively to the new one.
+-- If both tables have the same key, the second one takes precedence.
+-- @tparam table t1 The first table
+-- @tparam table t2 The second table
+-- @treturn table The (new) merged table
+-- @usage
+-- local t1 = {1, 2, 3, foo = "f"}
+-- local t2 = {4, 5, bar = "b"}
+-- local t3 = kong.table.merge(t1, t2) -- {4, 5, 3, foo = "f", bar = "b"}
+local function merge_tab(t1, t2)
+  local res = {}
+  if t1 then
+    for k,v in pairs(t1) do res[k] = v end
```

Comment:
note: we should avoid _all_ 1 liners, if we ever are to add back code coverage metrics, I believe our future selves will thank us.

Reply:
Ah yeah. It was a slip copied from old code